### PR TITLE
[9.4.x] ISPN-10784 xsite updates for 9.4.x concurrent writes

### PR DESCRIPTION
**Pull request by oraNod**
https://issues.redhat.com/browse/ISPN-10784

### CODE REVIEW

File: documentation/src/main/asciidoc/user_guide/cross_site_replication.adoc [hunk 1]
```diff
@@ -213,6 +213,47 @@ expired in clusters on other sites.
...
+
+[source,txt,options="nowrap"]
+----
+include::xsite_conflicts.txt[]
+----
+
+<1> `x` represents a key in the data store.
+<2> client writes `x=3` on "LON". "LON" replicates to "SFO".
+<3> client writes `x=5` on "SFO". "SFO" replicates to "LON"
+<4> client writes `x=6` on "LON" at the same time that a client writes `x=8` on "SFO".
```
Comment (wfink) [inline documentation/src/main/asciidoc/user_guide/cross_site_replication.adoc]: The sequence is not clear, it does not show when X is locked where.
I would have the sequence like
client write x=3 to LON
LON cache lock x
LON replicate to SFO and lock x(SFO)
......
For <4> you can see that
LON lock x (local)
SFO lock x (local)
LON repl x and try to lock x(SFO)
SFO repl x and try to lock x(LON) --> deadlock

File: documentation/src/main/asciidoc/user_guide/cross_site_replication.adoc [hunk 2]
```diff
@@ -213,6 +213,47 @@ expired in clusters on other sites.
...
+In asynchronous mode (`ASYNC`), the preceding example results in conflicts
+with one of the following outcomes:
+
+* `x` has a different value on "LON" than on "SFO". Asynchronous replication
+occurs after the client writes to `x` with the result that `x=8` is replicated
+to "LON" and `x=6` is replicated to "SFO".
+* `x` has the same value on "LON" and "SFO" because asynchronous replication
+overwrites one of the values, depending on when the transfer occurs. In this
+case, one of the values is lost and there is no guarantee which value is saved.
+
```
Comment (wfink) [inline documentation/src/main/asciidoc/user_guide/cross_site_replication.adoc]: I would mention that the inconsistence will be solved with the next non conflicting put

File: documentation/src/main/asciidoc/user_guide/cross_site_replication.adoc [hunk 3]
```diff
@@ -213,6 +213,47 @@ expired in clusters on other sites.
...
+To handle conflicts in asynchronous mode, client applications must implement a
+conflict resolution policy. For example, clients can randomly select which
```
Comment (wfink) [inline documentation/src/main/asciidoc/user_guide/cross_site_replication.adoc]: Is there a conflict resolution policy API available? I'm not aware of it by now.
If there link how to implement, if not mention future feature

File: documentation/src/main/asciidoc/user_guide/cross_site_replication.adoc (hunk 1 above)
Comment (oraNod) [reply documentation/src/main/asciidoc/user_guide/cross_site_replication.adoc]: @wfink I've updated. to me it makes it somewhat more confusing to try and show where the locks exist.

**Comment by pruivo** [comment]
integrated! thanks @oraNod !

**Pull request merged**